pkg/cvo/status.go

```diff
@@ -351,6 +349,26 @@ func (optr *Operator) syncStatus(ctx context.Context, original, config *configv1
 	return err
 }
 
+func setDesiredReleaseAcceptedCondition(config *configv1.ClusterVersion, status LoadPayloadStatus, now metav1.Time) {
+	if status.Step == "PayloadLoaded" {
+		resourcemerge.SetOperatorStatusCondition(&config.Status.Conditions, configv1.ClusterOperatorStatusCondition{
+			Type:               DesiredReleaseAccepted,
+			Status:             configv1.ConditionTrue,
+			Reason:             status.Step,
+			Message:            status.Message,
+			LastTransitionTime: now,
+		})
+	} else if status.Step != "" && status.Failure != nil {
+		resourcemerge.SetOperatorStatusCondition(&config.Status.Conditions, configv1.ClusterOperatorStatusCondition{
+			Type:               DesiredReleaseAccepted,
+			Status:             configv1.ConditionFalse,
```

Comment:
Is it worth trying to make some distinction between the "we're still working through this, but no serious issues yet, patience" where we'd go `Unknown`, and "last time we hit a fairly serious issue, but we're continually retrying, and maybe that issue will go away and this time we'll succeed" where we'd go `False`?